projects/optic/src/commands/oas/shapes/schema.ts

```diff
@@ -136,7 +136,7 @@ function initialSchema(
   rootInput: any,
   openAPIVersion: SupportedOpenAPIVersions
 ): OpenAPIV3.SchemaObject {
-  if (rootInput === null) {
+  if (rootInput === null || rootInput === undefined) {
```

Comment:
I think it's correct here to treat a keyed input but it returning `undefined` to be a `nullable` schema